plenum/common/ledger.py

```diff
@@ -0,0 +1,97 @@
+from copy import copy
+from typing import List, Tuple
+
+import base58
+
+from ledger.stores.chunked_file_store import ChunkedFileStore
+from ledger.stores.file_store import FileStore
+
+from ledger.ledger import Ledger as _Ledger
+
+
+class Ledger(_Ledger):
```

Comment:
Please use another name for a ledger supporting uncommitted txns, like BatchingLedger. Otherwise, if we have the same class names in different packages, there is always a risk that someone imports a wrong class by mistake.